internal/db/start/templates/_supabase.sql

```diff
@@ -0,0 +1,18 @@
+CREATE DATABASE _supabase WITH OWNER postgres;
+
+-- Switch to the newly created _supabase database
+\c _supabase
+-- Create schemas in _supabase database for
+-- internals tools and reports to not overload user database
+-- with non-user activity
+CREATE SCHEMA IF NOT EXISTS _realtime;
+ALTER SCHEMA _realtime OWNER TO postgres;
+CREATE SCHEMA IF NOT EXISTS realtime;
+ALTER SCHEMA realtime OWNER TO postgres;
```

Comment:
For some reason, if I don't include the `realtime` schema as well and only the `_realtime` the `initRealtime` function will fail requesting this other schema.